policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/tuition_amount/eligible_age.yaml

```
@@ -0,0 +1,10 @@
+description: British Columbia qualifies filers for the tuition credit of this age or above.
```

This source is for a federal program though. Does BC inherit the age limitation from Canada rules? Please add a legal code reference to check.
```suggestion
description: British Columbia limits the tuition credit to filers this age or older.
```

---

Should not be in this pr, deleted